sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -1077,6 +1077,44 @@ matrix(
               assert.equal(result.id, "100");
             });
 
+            it("should handle patchAsyncNoLocationHeader", async () => {
```

Comment:
This is the non-standard PATCH LRO, https://github.com/Azure/azure-sdk-assets/blob/js/eventgrid/arm-eventgrid_2193bddb25/js/sdk/eventgrid/arm-eventgrid/recordings/node/eventgrid_test/recording_domaintopiceventsubscriptions_update_test.json.